tests/conftest.py

```diff
@@ -206,6 +207,10 @@ def rand_one_dut_hostname(request):
         dut_hostnames = random.sample(dut_hostnames, 1)
     return dut_hostnames[0]
 
+@pytest.fixture(scope="module", autouse=True)
+def set_selected_tor_active(duthosts, rand_one_dut_hostname):
+    force_active_tor(duthosts[rand_one_dut_hostname], 'all')
```

Comment:
I don't see this method present in the dual_tor_utils file. I think using `toggle_simulator_port_to_upper_tor` would be more appropriate, since we don't want to explicitly change one of the ToR states to be active - the purpose of this fixture is just to ensure a consistent starting point for every test case.